### DISCUSSION TITLE
Replicating existing DB, error on startup: Unable to install change data capture pipeline

### DISCUSSION THREAD

**Discussion by hmhackmaster** [maxpert/marmot#83] [Q&A]
I am attempting to set up a secondary copy of Jellyfin (a network media player) to offload some specific components of the app. Among other things, Jellyfin uses two sqlite DBs (`jellyfin.db` and `library.db`).

For this use case, I have a 'primary' server and (currently) one secondary. If the primary isn't online I would not expect the secondary to work, so my goal was to use marmot to replicate each db between the two.
I haven't dug into the snapshot stuff yet as I kinda expected the secondary to 'catch up' from the primary db if the secondary was offline. (Each DB is currently less than 10MB). From what I understand, the snapshot would help make that faster, so I would probably do that down the line a bit (unless I am advised otherwise!).

After giving up on running the example (due to a port binding issue with NATS?), I decided to just try giving it a go. 

I have two issues so far; I am fully aware these are likely not issues with marmot itself, but I am hopeful that I might be able to gain some insight here.

1) for the `jellyfin.db`, when I start Marmot I get the following error:
`ERR Unable to install change data capture pipeline error="near \"Order\": syntax error" node_id=1`
To test, I completely nuked the Jellyfin install and started fresh with a brand new DB and got the same error, so I assume something wonky is happening inside their DB schema. Unfortunately, I am a sysadmin so I don't know where to look next. 

2) for `library.db`, I was able to get marmot running on my primary and secondary server. I ended up changing the Journal mode to WAL (to try and fix this issue) and saw that change appear on the secondary, so replication seems to work.
Unfortunately, when I then try to start Jellyfin, it complains that `Busy: SQLitePCL.pretty.SQLiteException: database is locked`.
If I start marmot second, marmot complains about a lock as well:
```
DBG Opening database node_id=1 path=/var/lib/jellyfin/data/library.db
ERR Unable to open database error="database is locked" node_id=1
```
To be honest, I didn't think db locking was going to be an issue based on my understanding of what marmot does, but perhaps I am going about this incorrectly.

Any thoughts are appreciated!

**Comment by maxpert** [comment]
I will try to set it up today and see if I can get it reproduced. Seems like bug in table creation or something where table name is order and I am not getting name quoted. Would you mind dumping the DB file here?
